Makefile

```diff
@@ -9,22 +9,24 @@ BASIC_TEST_PKGS := $(filter-out $(INTEGRATION_TEST_PKGS),$(TEST_PKGS))
 PACKAGES := go list ./... | grep -v 'dashboard/uiserver'
 PACKAGE_DIRECTORIES := $(PACKAGES) | sed 's|$(PD_PKG)/||'
 GOCHECKER := awk '{ print } END { if (NR > 0) { exit 1 } }'
-RETOOL := ./scripts/retool
 OVERALLS := overalls
 
-FAILPOINT_ENABLE  := $$(find $$PWD/ -type d | grep -vE "(\.git|\.retools)" | xargs ./scripts/retool do failpoint-ctl enable)
-FAILPOINT_DISABLE := $$(find $$PWD/ -type d | grep -vE "(\.git|\.retools)" | xargs ./scripts/retool do failpoint-ctl disable)
+GOBIN := $(shell pwd)/.gobin
+PATH := $(GOBIN):$(PATH)
+
+FAILPOINT_ENABLE  := $$(find $$PWD/ -type d | grep -vE "(\.git|\.retools)" | xargs failpoint-ctl enable)
```

Comment:
Can we remove all retools related things?

Reply:
It can be removed. But if an older version is used (retool is existed), is it need to skip these pkgs?

Reply:
Actually, there is no fail point in `.retools` packages.

Reply:
OK. And remote `.retools` in DEADLOCK_ENABLE and DEADLOCK_DISABLE?

Reply:
I think we can.

Reply:
There's no `vendor`.